spec/report_spec.rb

```diff
@@ -93,18 +95,5 @@
         expect(e.message).to eq('There are no more pages to retrieve.')
       end
     end
-
-    it 'reuses the params used to retrieve the first page' do
-      type = Fixture.report_type
-
-      first_page = described_class.all(
-        page_size: Fixture.page_size,
-        type: type,
-      )
-
-      next_page_params = described_class.build_next_page_params(first_page, first_page.reports, 0)
-
-      expect(next_page_params[:type]).to eq(type)
-    end
```

Comment:
issue: I don't think we wanted to remove this test? It's there to ensure we passed down the `type` to the response which is required to use this function

Reply:
This is the old helper function for using `get_next_page` which has been replaced by `get_next_page_helper` after consolidating, it's also the old unit test.